sound/pci/hda/patch_conexant.c

```diff
@@ -1121,6 +1121,7 @@ static int patch_conexant_auto(struct hda_codec *codec)
  */
 
 static const struct hda_device_id snd_hda_id_conexant[] = {
+	HDA_CODEC_ENTRY(0x14f120d0, "CX11970", patch_conexant_auto),
```

Comment:
I think this is the same as line 1128, isn't it? I'm guessing it got added upstream at some point. We might not need this specific commit any longer.

Reply:
Actually the combo jack also stopped working properly without those commits, so I bet we still need it. I'll double-check once the build finishes.